package.json

```diff
@@ -53,28 +53,28 @@
     "@improbable-eng/grpc-web": "^0.15.0",
     "@improbable-eng/grpc-web-fake-transport": "^0.15.0",
     "@types/google-protobuf": "^3.15.6",
-    "@typescript-eslint/eslint-plugin": "^5.57.1",
-    "@typescript-eslint/parser": "^5.57.1",
-    "@viamrobotics/eslint-config": "^0.0.3",
-    "@viamrobotics/prettier-config": "^0.0.1",
-    "@viamrobotics/typescript-config": "^0.0.3",
+    "@typescript-eslint/eslint-plugin": "^6.15.0",
+    "@typescript-eslint/parser": "^6.15.0",
+    "@viamrobotics/eslint-config": "^0.3.0",
+    "@viamrobotics/prettier-config": "^0.3.4",
+    "@viamrobotics/typescript-config": "^0.1.0",
     "concurrently": "^8.0.1",
     "copyfiles": "^2.4.1",
     "eslint": "^8.37.0",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-tsdoc": "^0.2.17",
```

Comment:
I don't see this plugin included in `.eslintrc.cjs`; is it used?

Reply:
Not sure -- I removed it and it didn't seem to cause any problems so.. I'll keep it out

Reply:
It looks like you removed `eslint-config-prettier` (which is used as a peer dep of `@viamrobotics/eslint-config`) instead of `eslint-plugin-tsdoc` (which looks like it isn't used). Was this intentional?

Ideally, we've found that `@viamrobotics/eslint-config` is easiest to maintain if _all_ its peer dependencies are specified. You can find a list of these in [its README](https://github.com/viamrobotics/js-config/blob/main/packages/eslint-config/README.md):

- @typescript-eslint/parser
- @typescript-eslint/eslint-plugin
- eslint
- eslint-config-prettier
- eslint-plugin-sonarjs
- eslint-plugin-unicorn

Without these peers specified, you run the risk of getting outdated peers even when you're on the latest version of our shared config (we keep the acceptable peer versions wide to minimize downstream disruption to config users)

Reply:
Ohhh that was a misunderstanding on my part -- I thought this comment was directed at the changed line, not at the tsdoc line 🤦‍♂️ Will fix